ixwebsocket/IXSelectInterruptFactory.cpp

```diff
@@ -20,7 +20,7 @@ namespace ix
 #if defined(__linux__) || defined(__APPLE__)
```

Comment:
I think we should reverse those ifdefs so that this code compiles on other unix (say freebsd etc...).

```
#ifdef _WIN32
#include "IXSelectInterruptEvent.h"
#else
#include "IXSelectInterruptPipe.h"
...
```